src/main/java/io/reactiverse/pgclient/PgPoolOptions.java

```diff
@@ -200,6 +205,7 @@ public PgPoolOptions setIdleTimeout(int idleTimeout) {
     return (PgPoolOptions) super.setIdleTimeout(idleTimeout);
   }
 
+  @Deprecated
```

Comment:
why this needs to be deprecated ?

Reply:
I believe we can let it as is and instead of having a boolean ssl, have it set the SSL mode to REQUIRE/DISABLE instead

Reply:
Updated with https://github.com/reactiverse/reactive-pg-client/pull/224/commits/905ee6008c187f760ce56dc0951d9c7da8e1cae2